locales/en.yml

```
@@ -3,7 +3,7 @@ en:
     description: "Add a tracking toggle to each category on the categories page"
     settings:
       tracking_level: "0: Muted, 1: Normal, 3: Watching first post, 4: Tracking, 5: Watching"
-      tracking_icon: '<a target="_blank" href="https://fontawesome.com/v5.15/icons?d=gallery&m=free">FontAwesome</a> icon name...'
+      tracking_icon: '<a target="_blank" href="https://fontawesome.com/search?o=r&m=free">FontAwesome</a> icon name...'
```

let's use the version-prefixed url, in case they release FA7 😃 

```suggestion
      tracking_icon: '<a target="_blank" href="https://fontawesome.com/v6/search?o=r&m=free">FontAwesome</a> icon name...'
```